src/Header.jsx

```diff
@@ -0,0 +1,74 @@
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogContentText,
+    DialogTitle,
+    Grid,
+    makeStyles,
+    Tooltip,
+    Typography
+} from '@material-ui/core';
+import InfoIcon from '@material-ui/icons/Info';
+import React from "react";
+
+const useStyles = makeStyles({
+  hccHeading: {
+    fontSize: '1.5rem',
+    '@media (min-width:280px)': {
+      fontSize: '2rem',
+    },
+    '@media (min-width:370px)': {
+      fontSize: '2.5rem',
+    },
+    '@media (min-width:600px)': {
+      fontSize: '3.5rem',
+    },
+  },
+});
+
+export default function Header (props) {
+    const classes = useStyles();
+
+    const [infoDisplayed, setInfoDisplayed] = React.useState(false);
+
+    const handleOpen = () => {
+        setInfoDisplayed(true);
+    };
+
+    const handleClose = () => {
+        setInfoDisplayed(false);
+    };
+    return (
+        <Grid container alignItems="center" justify="space-between" direction="row">
+            <Grid item xs={"auto"}>
+                <Typography variant="h1" gutterBottom className={classes.hccHeading}>{props.text}</Typography>
+            </Grid>
+            {props.info &&
+            <Grid item xs={false}>
+                <Tooltip
```

Comment:
Are you sure you want to keep both the tooltip and the dialog?

Reply:
I find it OK this way. Do you thing there should have a generic "About this app" tooltip instead?

Reply:
No, it's OK like it is, just wanted to make sure it's what you want.